Objects/abstract.c

```diff
@@ -733,6 +734,41 @@ int PyObject_CopyData(PyObject *dest, PyObject *src)
     return 0;
 }
 
+int
+PyObject_CopyToObject(PyObject *obj, void *buf, Py_ssize_t len,
+                      char Py_UNUSED(fort))
+{
+    Py_buffer view_obj;
+
+    if (!PyObject_CheckBuffer(obj)) {
+        PyErr_SetString(PyExc_TypeError,
+                        "destination must be bytes-like objects");
+        return -1;
+    }
+
+    if (!buf) {
+        PyErr_SetString(PyExc_MemoryError, "buf pointer is null");
+        return -1;
+    }
+
+    if (PyObject_GetBuffer(obj, &view_obj, PyBUF_SIMPLE)) {
+        return -1;
+    }
+
+    if (view_obj.len < len) {
+        PyErr_SetString(PyExc_BufferError,
+                        "destination is too small to"
+                        "receive data from buf");
```

Comment:
```suggestion
        PyErr_SetString(PyExc_BufferError,
                        "destination is too small to receive data from source");
```

Reply:
With your current example, you won't have a space between "to" and "receive", hence the suggestion.

Reply:
I have made change.